app.js

```diff
@@ -1,8 +1,124 @@
+import { renderPoll } from './render-utils.js';
+
 // import functions and grab DOM elements
+const optionAAddButton = document.getElementById('option-a-add');
+const optionAUndoButton = document.getElementById('option-a-undo');
+const optionBAddButton = document.getElementById('option-b-add');
+const optionBUndoButton = document.getElementById('option-b-undo');
+
+const form = document.querySelector('form');
+
+const submitPollButton = document.getElementById('submit-poll');
+const questionEl = document.getElementById('poll-question');
+const optionATitleEl = document.getElementById('option-a-title');
+const optionAVotesEl = document.getElementById('option-a-votes');
+const optionBTitleEl = document.getElementById('option-b-title');
+const optionBVotesEl = document.getElementById('option-b-votes');
+
+const pastPollsEl = document.getElementById('past-polls');
 
 // let state
+let question = '';
+let optionAVotes = 0;
+let optionBVotes = 0;
+let optionATitle = '';
+let optionBTitle = '';
+
+const pastPolls = [];
+
+optionAAddButton.addEventListener('click', () => {
+  //increment option A votes
+    optionAVotes++;
+
+
+    optionAVotesEl.textContent = optionAVotes;
```

Comment:
This works, but consider using your `displayCurrentPoll()` function since it does this same DOM updating and you don't have to write an extra line in each add or subtract event listener of something you already wrote in that function.